src/WCSimConstructRealisticPlacement.cc

```diff
@@ -0,0 +1,1168 @@
+    G4ThreeVector pmt_central_offset = G4ThreeVector(0.0,0.0,0.0); // -> Can be used for relative offset!
+    G4RotationMatrix* pmt_central_rotation = new G4RotationMatrix;
+    pmt_central_rotation->rotateX(90*deg);
+    pmt_central_rotation->rotateZ(270*deg);
+
+    G4RotationMatrix* multipmt_central_rotation = new G4RotationMatrix;
+    multipmt_central_rotation->rotateX(90*deg);
+    multipmt_central_rotation->rotateZ(270*deg);
+
+    // For the OD PMT we instead need to make it face outwards from the white tyvek in the OD.
+    G4ThreeVector odpmt_central_position = G4ThreeVector(config.WhiteTyvekOuterRadius+2*cm,0.0,0.0);
+    G4RotationMatrix* pmtod_central_rotation = new G4RotationMatrix;
+    pmtod_central_rotation->rotateY(270*deg);
+
+    auto pmt20_offset_placement = new G4AssemblyVolume();
+    pmt20_offset_placement->AddPlacedVolume(pmt20_dummy_logic, pmt_central_position, pmt_central_rotation);
+
+    auto pmtmulti_offset_placement = new G4AssemblyVolume();
+    pmtmulti_offset_placement->AddPlacedVolume(pmtmulti_dummy_logic, mpmt_central_position, multipmt_central_rotation);
+
+    auto pmtod_offset_placement = new G4AssemblyVolume();
+    pmtod_offset_placement->AddPlacedVolume(pmtod_dummy_logic, odpmt_central_position, pmtod_central_rotation);
+
+
+    // PMT_only_two_cell/PMT_hybrid_three_cell/PMT_od_central_cell
+    // ************************
+
+    // Now our job is to make a cell. The following diagrams show locations (#,O,M,F) = (None, 20inch, multiPMT, OD)
+    //
+    // PMT_only_two_cell
+    // _______
+    // |     |
+    // | # O |
+    // |     |
+    // | O # |
+    // _______
+
+    // PMT_hybrid_three_cell
+    // _______
+    // |     |
+    // | # O |
+    // |     |
+    // | O M |
+    // _______
+
+    // PMT_od_central_cell
+    // _______
+    // |     |
+    // | # # |
+    // |     |
+    // | #M# |
+    // _______
+
+    G4double RowSeperation = config.RowSeperation;
+
+    int NSpacesInBlock = config.NSpacesInBlock;
+    int NSegments = config.NBlocksAround * NSpacesInBlock;
+    int NBlocksUp = config.NBlocksUp;
+    G4double phi_offset = twopi / float(NSegments);
+
+    std::cout << "PHI OFFSET : " << phi_offset/2 << std::endl;
+
+    // First we build PMT_only_two_cell
+    G4RotationMatrix* rotation_low = new G4RotationMatrix;
+    G4RotationMatrix* rotation_high = new G4RotationMatrix;
+
+    rotation_high->rotateZ(phi_offset/2); //-phi_offset/2);
+    rotation_low->rotateZ(3*phi_offset/2);//align the PMT with the Cell
+
+    G4ThreeVector position_low = G4ThreeVector(0,0.0,-RowSeperation/4);
+    G4ThreeVector position_high = G4ThreeVector(0,0.0,RowSeperation/4);
+
+    auto frame_block_assembly = new G4AssemblyVolume();
+    frame_block_assembly->AddPlacedAssembly(pmt20_offset_placement, position_high, rotation_high );
+    // frame_block_assembly->AddPlacedAssembly(pmtmulti_offset_placement, position_high, rotation_low );
```

Comment:
```suggestion
    // skip the top left position in the cell, as it is empty (position_high, rotation_low)
```
I like the idea you have with the empty parts having a placeholder, but prefer to have a comment instead of commented out code

Choose your favourite wording & then please repeat it for all similar lines